packages/e2e-tests/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@sentry-internal/e2e-tests",
+  "version": "0.0.0",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "test:e2e": "ts-node run.ts"
```

Comment:
Can we add commands prettier/eslint in another PR?

Reply:
Added here: 25335fb24e1b8a9b27cc7eede8021088c019c9bd